content/html/concepts/favicon/favicon.md

```diff
@@ -0,0 +1,27 @@
+---
+Title: 'Favicon'
+Description: "A favicon is a small icon that is displayed in the browser's address bar and tab title for a website."
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Favicon'
+  - 'Documentation'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+A **favicon** (short for "favorite icon") is a small icon that is displayed in the browser's address bar and tab title for a website. It is used to help users quickly identify the website and distinguish it from other tabs or bookmarks.
+
+To add a favicon to your website, you need to create a small image that will be used as the favicon, and then include a link to it in the head of your HTML document using the `link` element. Here's an example of how to do this:
+
+```html
+<link rel="shortcut icon" type="image/x-icon" href="/path/to/favicon.ico" />
+```
+
+The `link` element with the `rel` attribute set to "shortcut icon" tells the browser that the linked resource is a favicon. The `type` attribute specifies the type of the favicon (in this case, "image/x-icon"), and the `href` attribute specifies the location of the favicon file.
```

Comment:
I think this would work well as an unordered list ^^

```suggestion
- The `link` element with the `rel` attribute set to "shortcut icon" tells the browser that the linked resource is a favicon.
- The `type` attribute specifies the type of the favicon (in this case, "image/x-icon").
- The `href` attribute specifies the location of the favicon file.
```

Reply:
I think the first bullet point should only cover `rel` because this is what tells the browser that the linked resource is a favicon. Feel free to include a bullet point about the `<link>` element though and link our existing entry: https://www.codecademy.com/resources/docs/html/elements/link

Reply:
cc: @Dusch4593 This gives an idea to create terms for [HTML attributes](https://www.codecademy.com/resources/docs/html/attributes)

Reply:
@yangc95 Hmmmmm, I can see there being terms for _global HTML attributes_ such as `autocapitalize` and `dir`. What do we think?

Reply:
@Dusch4593 @SSwiniarski Continued discussion here: https://github.com/Codecademy/docs/discussions/1468